src/service/resources-impl.js

```diff
@@ -169,6 +173,7 @@ export class Resources {
     });
     this.viewport_.onScroll(() => {
       this.lastScrollTime_ = timer.now();
+      this.framerate_.collect();
```

Comment:
You mean flush at the end of the collection phase? If so, I think yes - it's just once per 5 seconds or so.